modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -43,7 +43,6 @@ public class {{classname}}{{#parent}} extends {{{parent}}}{{/parent}}{{^parent}}
     {{/isContainer}}
     {{/isEnum}}
   {{#jackson}}
-  @JsonProperty("{{baseName}}")
   {{#withXml}}
```

Comment:
You may want some unit tests that validate where this property is being written - see something like `shouldGenerateDiscriminatorFromAllOfWhenUsingLegacyDiscriminatorBehaviour_issue12692`